signalfx-org-metrics/metrics.yaml

```diff
@@ -1290,6 +1290,17 @@ sf.org.numPropertyLimitedMetricTimeSeriesCreateCallsByToken:
   metric_type: counter
   title: sf.org.numPropertyLimitedMetricTimeSeriesCreateCallsByToken
 
+sf.org.numResourceMetrics:
```

Comment:
Does our user documentation already have a definition of "Resource metric time series"? I just added a PR to explain the terms "active" MTS and "inactive MTS", and I anticipate that someone is going to ask for the same explanation of Resource metric time series.

Reply:
Yes -- a "resource MTS" represents the use of subscription resources like custom MTS.